<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMirror</class>
 <widget class="QDialog" name="NewMirror">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Mirror</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Foucault2.qrc">
    <normaloff>:/Foucault2/Foucault2.ico</normaloff>:/Foucault2/Foucault2.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Optical Diameter (mm)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Focal Length (mm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leOpticalDiameter"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Conical constant</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leConicalConstant">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leFocal"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hole diameter (mm)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leHoleDiameter">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Couder Screen</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Slit type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSlitType">
          <item>
           <property name="text">
            <string>Unknown</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Slit is Still</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Slit is moving</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Hx (mm):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="twHx">
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>10</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Manual Design</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Design with number of zones</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leNumberOfZones">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnComputeNbOfzone">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Compute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>Design with nb millimeters by zone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMillimetersByZone">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>1.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnComputeNbMillimetersByZone">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Compute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>Design with min zone size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMinZoneSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>10.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnComputeMinZoneSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Compute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leOpticalDiameter</tabstop>
  <tabstop>leHoleDiameter</tabstop>
  <tabstop>leFocal</tabstop>
  <tabstop>leConicalConstant</tabstop>
  <tabstop>twHx</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="Foucault2.qrc"/>
 </resources>
 <connections/>
</ui>
